en/common_voice_privacy_notice.md

```diff
@@ -1,16 +1,16 @@
 # Common Voice Privacy Notice 
 
-Effective June 27, 2024 {: datetime="2024-06-27" }
+Effective November 4, 2024 {: datetime="2024-11-04" }
 
 ## Privacy
 
 When Mozilla (that’s us), receives information from you, our [Mozilla Privacy Policy](https://www.mozilla.org/privacy) describes how we handle that information.
 
 * **Demographic data.** You can optionally send us information such as your accent, age, and gender. This helps us and other researchers improve and create speech-to-text technology and tools.
 
-* **Account data.** You do not need to create an account to use Common Voice. If you decide to create an account, we receive your username and avatar, if you submit one. Your email address is associated with your demographic and interaction data but is not shared to the public. We display leaderboards showing the number of recordings users make. You have the option whether or not you wish to appear on the leaderboards. You can delete your account at any time and your username and email will be removed.
+* **Account data.** You do not need to create an account to use Common Voice. If you decide to create an account, you will need to create a Mozilla account [here](commonvoice.mozilla.org/mozaws.net/signin), and we receive your username and avatar, if you submit one. Your use of Mozilla accounts is governed by the [Mozilla Accounts Terms of Service](https://www.mozilla.org/about/legal/terms/services/) and [Mozilla Accounts Privacy Notice](https://www.mozilla.org/privacy/mozilla-accounts/). Your email address is associated with your demographic and interaction data but is not shared to the public. We display leaderboards showing the number of recordings users make. You have the option whether or not you wish to appear on the leaderboards. You can delete your account at any time and your username and email will be removed. You can delete your Mozilla aaccount at any time and your username and email will be removed.
```

Comment:
`commonvoice.mozilla.org/mozaws.net/signin` is missing https://, but it's also a broken link (404 page).

Reply:
Can we update to https://commonvoice.mozilla.org/mozaws.net/signin ? 
It's an Error Page currently, but it will be live on November 4. Is that OK?

Reply:
We will make sure the doc goes live on Nov 4th, not earlier.